bowling/src/frameParser.cpp

```diff
@@ -0,0 +1,40 @@
+#include <algorithm>
+#include <functional>
+
+#include "frameParser.hpp"
+#include "tokenizer.hpp"
+
+const char FrameParser::StrikeSign = 'X';
+const char FrameParser::MissSign = '-';
+const char FrameParser::SpareSign = '/';
+
+
+size_t FrameParser::parseToken(const char token, size_t& lastParsedToken) {
+    if (token == FrameParser::StrikeSign) {
+        lastParsedToken = 10;
+        return lastParsedToken;
+    } else if (token == FrameParser::SpareSign) {
+        lastParsedToken = 10 - lastParsedToken;
+        return lastParsedToken;
+    } else if (token == FrameParser::MissSign) {
+        lastParsedToken = 0;
+        return lastParsedToken;
+    } else {
+        lastParsedToken = token - '0';
```

Comment:
Could you be more specific? Like static_cast<size_t> ? This will not produce needed output.

Reply:
Why not? I've just checked `lastParsedToken = static_cast<size_t>(token - '0');` - it compiles and tests still pass.

I'm not sure if it's better to use implicit or explicit casting, but based on experience with other languages I believe it's better to avoid implicit type conversion.

Reply:
Done.